### ISSUE TITLE
Unexpected WordPress.WP.AlternativeFunctions.file_system_read_fopen warning

### ISSUE THREAD

**Issue by paulschreiber** [WordPress/WordPress-Coding-Standards#1173]
[PHPCS 3.1, WPCS 0.13.1, PHP 5.6.30]

I'm using `fopen()` with a stream, and WPCS is now flagging this as filesystem usage:
```php
<?php
Header( 'HTTP/1.1 200 OK' );
Header( 'Content-Type: text/csv' );
Header( "Content-Disposition: attachment; filename={$filename}" );

$output_stream = fopen( 'php://output', 'w' );

fputcsv( $output_stream, $output_columns );
foreach ( self::$metadata as $row ) {
	fputcsv( $output_stream, $row );
}

fclose( $output_stream );
```

I get this error:
>File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: fopen()

WP_Filesystem doesn't seem appropriate here.

Do you recommend a different way of writing the above code?

**Comment by JDGrimes** [comment]
This is related to #295, though that is for a different sniff.

**Comment by GaryJones** [comment]
Very relevant: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1265#issuecomment-405143028

**Comment by jrfnl** [comment]
Closing as fixed by #1655 which was included in WPCS 2.1.0.
